src/main/java/com/marklogic/kafka/connect/source/TimestampKeyGenerator.java

```diff
@@ -0,0 +1,11 @@
+package com.marklogic.kafka.connect.source;
+
+import java.time.Instant;
+
+public class TimestampKeyGenerator implements KeyGenerator {
+
+    @Override
+    public String generateKey(Long rowNumber) {
+        return Long.toString(Instant.now().toEpochMilli()).concat("-").concat(Long.toString(rowNumber));
```

Comment:
Is this the same as `System.currentTimeInMillis()`? 

@BillFarber  It occurs to me now that perhaps the ML server timestamp would be more helpful here. The reason being that I think it would be preferable if each of these keys have the exact same timestamp, with only the row number differing. And the key now has some meaning - it's tied to something in MarkLogic (though that something - the server timestamp - unfortunately is not durable and will disappear at some point in the future).

Reply:
I'm fine with using the server timestamp.  It would help identify batches ... if that would be of any use.

Reply:
@rahulvudutala I think this works for now, I'll take a look later on to see if we can use the server timestamp instead.